### ISSUE TITLE
Invisible videos

### ISSUE THREAD

**Issue by Archeix7** [libre-tube/LibreTube#1757]
### Steps to reproduce

I have repeatedly noticed that some videos that have already been published are not displayed in the "Subscriptions", only after some time they are wedged between other videos that have already been released.
 It also happens if you change the Instance, that the videos that are already available with another instance are not in this one.

1. Save your subscription file
2. Change a couple of instances and upload a subscription file to each of them
3. Result

(There is also a feeling that some channels, with some instances, will never be displayed, requires verification)

### Expected behavior

So that when a video is released, it will be displayed in subscriptions

### Actual behavior

.

### LibreTube version

0.7.0

### Android version

А11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
see https://github.com/TeamPiped/Piped/issues/1130
